python/ccf/ledger.py

```diff
@@ -169,13 +180,42 @@ def __next__(self):
         except:
             raise StopIteration()
 
-
 class Ledger:
 
-    _filename = None
+    _filenames = []
+    _directory = None
+    _fileindex = 0
 
-    def __init__(self, filename):
-        self._filename = filename
+    def __init__(self, name:str, is_dir:bool == False):
```

Comment:
I think it's OK for `name` to always be a directory here. The directory contains one or multiple ledgers that can be parsed with this class.

Reply:
Removed the parameter itself, just checking now if os.path.isdir(name)